native/libcst/benches/parser_benchmark.rs

```diff
@@ -118,9 +119,20 @@ pub fn parse_into_cst_benchmarks<T: Measurement>(c: &mut Criterion<T>) {
     group.finish();
 }
 
+#[cfg(target_arch = "x86_64")]
+fn get_config() -> Criterion {
+    // criterion_cycles_per_byte is only supported on x86
+    Criterion::default().config.with_measurement(CyclesPerByte)
```

Comment:
That `config` field is private, apparently:
```
❯ cargo test --bench parser_benchmark
   Compiling libcst v0.1.0 (C:\Users\zsolz\projects\libcst\native\libcst)
error[E0616]: field `config` of struct `Criterion` is private
   --> libcst\benches\parser_benchmark.rs:125:26
    |
125 |     Criterion::default().config.with_measurement(CyclesPerByte)
    |                          ^^^^^^ private field

For more information about this error, try `rustc --explain E0616`.
error: could not compile `libcst` due to previous error
```

Reply:
My bad! Fixed